docs/index.md

```diff
@@ -2,10 +2,25 @@
 
 ---
 
-The documentation on this site is written for developers.
-People who want to contribute to Decred, or create their own Decred based applications.
+If you're looking to develop Decred-based applications, become a 
+contributor, or simply gain a deeper understanding of Decred, 
+this is where you'll find technical documentation such as specifications,
+deeper dives into core concepts, how to set up a development environment
+and guidelines for contributing.
+
+Note that Decred's documentation is, like the project, decentralized. This
+site aims to present all the documentation relevent to all projects. However
+important documentation for some projects lives on GitHub, where developers
+working on the project can more easily maintian it. For instance, much of the
+documentation for dcrd, Decred's node daemon, lives at [dcrd/docs](https://github.com/decred/dcrd/tree/master/docs).
+
+User documentation can be found [here](https://docs.decred.org).
```

Comment:
i hate clicking on "here".

Reply:
Was trying to avoid here, but couldn't think of a better way. Linking to [docs.decred.org](https://docs.decred.org) as you suggest works better. Going with that.